ep/src/lib.rs

```diff
@@ -64,9 +71,9 @@ pub struct Epaddr <'a> {
 
 
 impl Ep {
-  // TODO: change job_key to rust uuid, add psm_ep, add new
-  pub fn open<'a>(job_key: u64, ep_opts: EpOpts) -> Result<(Ep, &'a Epaddr<'a>), Error> {
-    Err(Error::new(ErrorType::PSM_ERROR_LAST, "send help"))
+  // TODO: add psm_ep, add new
+  pub fn open<'a>(job_key: uuid::Uuid, ep_opts: EpOpts) -> Result<(Ep, &'a Epaddr<'a>), Error> {
+    Err(Error::PSM_ERROR_LAST)
   }
 
   pub fn close(ep: Ep, mode: isize, timeout: u64) -> Result<(), Error> {
```

Comment:
If your result is just an error, maybe consider an Option<Error> instead? Its a bit backwards from what I'd usually do too. Just throwing it out -- don't get stuck in the C mindset.

Again, either way is fine.